cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/SegmentStoreCommand.java

```diff
@@ -14,17 +14,22 @@
  * limitations under the License.
  */
 package io.pravega.cli.admin.segmentstore;
-
 import io.pravega.cli.admin.AdminCommand;
 import io.pravega.cli.admin.CommandArgs;
+import io.pravega.controller.server.security.auth.GrpcAuthHelper;
 
 /**
  * Base class for all the Segment Store related commands.
  */
 public abstract class SegmentStoreCommand extends AdminCommand {
     static final String COMPONENT = "segmentstore";
+    protected final GrpcAuthHelper authHelper;
 
     public SegmentStoreCommand(CommandArgs args) {
         super(args);
+
+        authHelper = new GrpcAuthHelper(true,
+                "secret",
```

Comment:
What is the purpose of these hardcoded values? Would we ever want to change them at runtime?

Reply:
@anirudhkovuru mentioned that this value should not be left for the user to manipulate, because its a secure value that should be consistent. So, I took it off from the parameters we pass in for the cli. You can correct me if I missed anything Anirudh, ty.

Reply:
In this case, `"secret"` refers to the `tokenSigningKey` which is a server secret that is used for controller and segmentstore communications.

Reply:
I see; so it's not actually the secret, rather the key that use used to get the actual secret from the environment, correct?